src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -53,19 +51,17 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
       );
 
       test(
-        'split panel focus moves to slider on open and open button on close',
+        'split panel focus moves to slider on open, and open button on close',
         setupTest(
           async page => {
-            const splitPanelOpenActionEl =
-              theme === 'refresh-toolbar'
-                ? wrapper.findDrawerTriggerById('slide-panel').toSelector()
-                : wrapper.findSplitPanel().findOpenButton().toSelector();
-            await page.click(splitPanelOpenActionEl);
```

Comment:
The `findOpenButton()` utility works in both versions, no need for separate selectors.